{"name": "新增用户", "status": "passed", "attachments": [{"name": "log", "source": "6fa2482a-d352-4db8-b034-6d9ff883150f-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "inData", "value": "'{\"password\":\"\",\"password_confirm\": \"88888888\",\"username\":\"001\"}'"}, {"name": "exp", "value": "'{'code':500,'msg':\"Key:'userAddRequest.password'Error:Field validation for 'password' failed on the'required' tag\"}'"}], "start": 1605080496199, "stop": 1605080496230, "uuid": "78d84d89-8f24-4285-b019-b0394cc5afd9", "historyId": "ecb303ec57313fc1864e58ab91f4046c", "testCaseId": "dc3f5c27b8f716b793706de44b0c3c3f", "fullName": "test_UserManage.TestUserManage#test_add_user", "labels": [{"name": "suite", "value": "test_UserManage"}, {"name": "subSuite", "value": "TestUserManage"}, {"name": "host", "value": "dadaxPC008"}, {"name": "thread", "value": "1252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_UserManage"}]}